assets/src/edit-story/components/panels/index.js

```diff
@@ -79,19 +79,17 @@ export function getPanels(elements) {
 
   // Only display background panel in case of background element.
   if (isBackground) {
-    const panels = [
-      { type: BACKGROUND_OVERLAY, Panel: BackgroundOverlayPanel },
-    ];
-    // If the selected element's type is video / image , display accessibility panel, too.
-    if ('shape' === elements[0].type) {
-      panels.unshift({ type: SHAPE_STYLE, Panel: ShapeStylePanel });
```

Comment:
This panel should always be shown (that's partoof #1177) – and we need to work on what it does when there's a background image.

We essentially need to move the background color from the (default) background element to the page.

I can't remember if I'm doing that in any of the existing background PR's (don't think so), so I think it needs to happen here - with migration and everything.

Reply:
Oops, I looked at the incorrect Figma for when nothing is selected. Thanks! Will re-check.